.travis.yml

```diff
@@ -0,0 +1,28 @@
+dist: trusty
+sudo: required
+
+language: go
+go: '1.12.x'
+
+services:
+  - docker
+
+install:
+  - GOPATH=${TRAVIS_BUILD_DIR}/vendor/go
```

Comment:
Why we do need to modify the gopath here?

Reply:
Because the build with goreleaser needs go modules. As it happens in the deploy step we need to preserve those file in the build dir.